### ISSUE TITLE
Implement tf.keras.layers.BatchNormalization

### ISSUE THREAD

**Issue by mortendahl** [tf-encrypted/tf-encrypted#515]
Leftover from https://github.com/tf-encrypted/tf-encrypted/issues/479
